pybox/inject.py

```diff
@@ -36,5 +37,9 @@ def __get__(self, instance: Optional[IService], owner: Type[IService]) -> IServi
         if instance is None:
             raise InstanceIsRequiredException()
         if self._attribute_name not in instance.__dict__:
-            instance.__dict__[self._attribute_name] = Container.get(self._dependency)
+            instance.__dict__[self._attribute_name] = Container().get(self._dependency)
         return instance.__dict__[self._attribute_name]
+
+
+def inject(dependency: Type[T]) -> T:
```

Comment:
`inject` injects dependency eager, while `Inject` does it lazy. I cannot image the real needs of this functionality, but who knows